worlds/undertale/docs/en_Undertale.md

```diff
@@ -56,8 +56,8 @@ If you press `W` while in the save menu, you will teleport back to the flower ro
 The following commands are only available when using the UndertaleClient to play with Archipelago.
 
 - `/resync` Manually trigger a resync.
-- `/patch` Patch the game.
-- `/savepath` Redirect to proper save data folder. (Use before connecting!)
+- `/patch` Patch the game. Only use this command if `/autopatch` fails.
```

Comment:
This should probably move below `/auto_patch`

Reply:
[ce41e4a2](https://github.com/ArchipelagoMW/Archipelago/pull/2435/commits/c41e4a274d1227776c671d87cb6884e847d446f7)